gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-8.1.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.2.1-bin.zip
```

Comment:
One other question: any particular dependency/feature that motivated this version bump? I usually try to lag behind on Gradle versions a bit in case the toolchain hasn't been tested with it.

Reply:
I recommend you check these two links seperately:

[Gradle 8.2 Release Notes](https://docs.gradle.org/8.2/release-notes.html),
[Gradle 8.2.1 Release Notes](https://docs.gradle.org/8.2.1/release-notes.html).